test/simulator/timestamp/src/session_simulator.cpp

```diff
@@ -165,9 +165,19 @@ session_simulator::commit_transaction(const std::string &config)
 
     auto pos = config_map.find("commit_timestamp");
     if (pos != config_map.end()) {
-        WT_SIM_RET(ts_manager->validate_hex_value(pos->second, "commit timestamp"));
+        int ret = ts_manager->validate_hex_value(pos->second, "commit timestamp");
+        if (ret != 0) {
+            rollback_transaction("");
```

Comment:
Should we allow the default configuration to be an empty string? Then we could just do `rollback_transaction();`

Reply:
Updated in [dee65cd](https://github.com/wiredtiger/wiredtiger/pull/8372/commits/dee65cde079269b2222f4d550ad37663a7a5767e)